docs/HACKING.md

```diff
@@ -55,10 +55,18 @@ Add the following lines to your shell startup script (`.bashrc`, `.zshrc`, etc.)
     export PROJECT_HOME=$HOME/devel
     source /usr/local/bin/virtualenvwrapper.sh
 
+Wherever you add these flags, ensure to source it on all your open terminal windows. On most MAC OS's the ".bashrc" file does not exist in the "~/.bashrc" path. You might be able to find equivalent files like "/etc/bash_profile" or "/etc/profile". You can also create your own "/etc/bashrc" file. If you do create your own file, add these lines to your bash_profile: 
+
+    if [ -f ~/.bashrc ]; then
+        . ~/.bashrc
+    fi 
+
```

Comment:
I have mine .bash_profile like this on my mac just to show a few additional things you may need :) 


if [ -f /opt/local/etc/profile.d/bash_completion.sh ]; then
    . /opt/local/etc/profile.d/bash_completion.sh
fi

p2=/opt/local/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/site-packages/
PATH=/usr/local/bin:${PATH/:\/usr\/local\/bin}:/usr/texbin:${p2}

if [ -f ~/.bashrc ]; then
          source ~/.bashrc
        else
       echo "No .bashrc file"
fi




> On Dec 1, 2015, at 15:17, bhuvan-rk <notifications@github.com> wrote:
> 
> In docs/HACKING.md <https://github.com/Solinea/goldstone-server/pull/145#discussion_r46355100>:
> 
> > @@ -55,10 +55,18 @@ Add the following lines to your shell startup script (`.bashrc`, `.zshrc`, etc.)
> >      export PROJECT_HOME=$HOME/devel
> >      source /usr/local/bin/virtualenvwrapper.sh
> >  
> > +Wherever you add these flags, ensure to source it on all your open terminal windows. On most MAC OS's the ".bashrc" file does not exist in the "~/.bashrc" path. You might be able to find equivalent files like "/etc/bash_profile" or "/etc/profile". You can also create your own "/etc/bashrc" file. If you do create your own file, add these lines to your bash_profile: 
> > +
> > +    if [ -f ~/.bashrc ]; then
> > +        . ~/.bashrc
> > +    fi 
> > +
> I did not have one. All of my settings are sourced from /etc/bash_profile. Some very long stackoverflow threads exist explaining the reason why.
> 
> Also note the words "On most MAC OS's" :)
> 
> —
> Reply to this email directly or view it on GitHub <https://github.com/Solinea/goldstone-server/pull/145/files#r46355100>.
>